astronomer/cleanup_pods/command_line.py

```diff
@@ -1,9 +1,12 @@
 import argparse
 import logging
+import os
 
 from kubernetes import client, config
 from kubernetes.client.rest import ApiException
 
+from datetime import datetime, timezone
+
```

Comment:
```suggestion
from datetime import datetime, timezone

from kubernetes import client, config
from kubernetes.client.rest import ApiException

```

ruff wants to reorder these imports